### PR TITLE
Use a fixed version of CLI

### PR DESCRIPTION
**Pull request by pranavkm**
cc @anurse

### CODE REVIEW

**Comment by anurse** [comment]
:ship: :it:

File: KoreBuild-dotnet/build/KoreBuild.cmd [hunk 1]
```diff
@@ -25,7 +25,7 @@ IF NOT EXIST xunit.core  (
 SET DOTNET_LOCAL_INSTALL_FOLDER=%LOCALAPPDATA%\Microsoft\dotnet\cli
 SET DOTNET_LOCAL_INSTALL_FOLDER_BIN=%DOTNET_LOCAL_INSTALL_FOLDER%\bin
 
-CALL %~dp0dotnet-install.cmd
+CALL %~dp0dotnet-install.cmd -version "1.0.0.000973"
```
Comment (davidfowl) [inline KoreBuild-dotnet/build/KoreBuild.cmd]: Env variable? so we can control it from the CI as well?

Comment (davidfowl) [inline KoreBuild-dotnet/build/KoreBuild.cmd]: What about the .sh

Comment (pranavkm) [inline KoreBuild-dotnet/build/KoreBuild.cmd]: I'll follow up on this.

**Pull request closed without merging**